docs/source/upcoming_release_notes/1058-Add_new_motors_into_spectrometer.rst

```diff
@@ -0,0 +1,30 @@
+1058 Add new motors into spectrometer
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- Add two new thorlabs ZST213 into TMOSpectrometer
+
+Device Updates
+--------------
+- Add Thorlabs ZST213
+
+New Devices
+-----------
+- Thorlabs ZST 213 x 2
+
+Bugfixes
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- tongju@slac.stanford.edu
```

Comment:
These are miscategorized, I'll push this change and see if it causes the gh to update properly.

```suggestion
Features
--------
- N/A

Device Updates
--------------
- Add two new thorlabs ZST213 into TMOSpectrometer, lens_pitch and lens_yaw.

New Devices
-----------
- N/A

Bugfixes
--------
- N/A

Maintenance
-----------
- N/A

Contributors
------------
- tongju12
```